configs/dnnsoftware.json

```diff
@@ -1,25 +1,25 @@
 {
   "index_name": "dnnsoftware",
   "sitemap_urls": [
-    "https://dnndocs.com/sitemap.xml"
+    "https://docs.dnncommunity.org/sitemap.xml"
   ],
   "start_urls": [
     {
-      "url": "https://dnndocs.com/api/",
+      "url": "https://docs.dnncommunity.org/api/",
       "selectors_key": "api",
       "tags": [
         "api"
       ]
     },
     {
-      "url": "https://dnndocs.com/api/DotNetNuke.Common.Lists.ListInfoCollection.html",
+      "url": "https://docs.dnncommunity.org/api/DotNetNuke.Common.Lists.ListInfoCollection.html",
       "selectors_key": "api",
       "tags": [
         "api"
       ]
     },
```

Comment:
Only the object for the URL `https://docs.dnncommunity.org/api/` should be needed since they are similar